.github/workflows/release.yml

```diff
@@ -35,6 +35,7 @@ jobs:
         uses: python-semantic-release/python-semantic-release@v8.0.4
```

Comment:
I would generally not force it especially since we use the squash-merge it can all be handles during the review. We want to make it as easy as possible to commit.

Reply:
Okay then 🙂